sql/migrations/20210731105804_world.sql

```diff
@@ -0,0 +1,93 @@
+DROP PROCEDURE IF EXISTS add_migration;
+delimiter ??
+CREATE PROCEDURE `add_migration`()
+BEGIN
+DECLARE v INT DEFAULT 1;
+SET v = (SELECT COUNT(*) FROM `migrations` WHERE `id`='20210731105804');
+IF v=0 THEN
+INSERT INTO `migrations` VALUES ('20210731105804');
+-- Add your query below.
+
+
+-- Father Inigo Montoy Gossip
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(7138, 8400);
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `option_broadcast_text`, `option_id`, `npc_option_npcflag`, `action_menu_id`) VALUES
+(7098, 0, 0, 'Tell me about faith, Father.', 12045, 1, 1, 7138);
+
+-- Commander Eligor Dawnbringer Gossip
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(7110, 8367);
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `option_broadcast_text`, `option_id`, `npc_option_npcflag`, `action_menu_id`) VALUES
+(7097, 0, 0, 'What is Redemption armor?', 11976, 1, 1, 7110);
+
+-- Mataus the Wrathcaster Gossip
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(7136, 8399),
+(7137, 8398);
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `option_broadcast_text`, `option_id`, `npc_option_npcflag`, `action_menu_id`) VALUES
+(7105, 0, 0, 'What is Plagueheart armor?', 12041, 1, 1, 7137),
+(7137, 1, 0, 'I also seek such power.', 12043, 1, 1, 7136);
+
+-- Jessica Chambers Gossip
+INSERT INTO `gossip_menu` (`entry`, `text_id`) VALUES
+(6064, 7180);
+INSERT INTO `gossip_menu_option` (`menu_id`, `id`, `option_icon`, `option_text`, `option_broadcast_text`, `option_id`, `npc_option_npcflag`, `action_menu_id`) VALUES
+(7173, 3, 0, 'Tell me about dungeons I could explore.', 9882, 1, 1, 6064),
+-- NO ACTION MENU ID AVAILABLE

```

Comment:
That part is the same for all innkeepers, just use the existing menu with the dungeon options.